src/lib/filters/scraps/scrapPing.go

```diff
@@ -26,12 +26,25 @@ func CLIPingScrap(s *string) (a *TypePingScrap) {
 		if words[0] == "rtt" {
 			temp := strings.Split(words[len(words)-2], "/")
 			a = &TypePingScrap{
-				Min: strToFloat64(temp[0]),
-				Avg: strToFloat64(temp[1]),
-				Max: strToFloat64(temp[2]),
+				Min:  strToFloat64(temp[0]),
+				Avg:  strToFloat64(temp[1]),
+				Max:  strToFloat64(temp[2]),
 				Mdev: strToFloat64(temp[3]),
 			}
 		}
 	}
 	return
-}
\ No newline at end of file
+}
+
+// CLIFLoodPingScrap returns packet loss
+func CLIFLoodPingScrap(s *string) (a uint64) {
+	arr := strings.Split(*s, "\n")
+	sentence := arr[3]
+	words := strings.Split(sentence, ", ")
+	percentage := strings.Split(words[2], " ")[0]
+	a, err := strconv.ParseUint(strings.TrimRight(percentage, "%"), 10, 32)
```

Comment:
it would be really helpful if we could get ***min/avg/max/mdev*** values of ping along with the packet loss rather than just the packet loss. This is due to the fact that sending data in a flooded environment will have a significant impact on the ping as well, thus denoting a real-world scenario. Please change this accordingly.